packages/react/src/utils/questionnaire.ts

```diff
@@ -389,28 +389,39 @@ export function getQuestionnaireItemReferenceFilter(
   return result;
 }
 
-export function buildInitialResponse(questionnaire: Questionnaire): QuestionnaireResponse {
+export function buildInitialResponse(
+  questionnaire: Questionnaire,
+  questionnaireResponse: QuestionnaireResponse | undefined
```

Comment:
Maybe make this optional param?  Should be same behavior, but with the benefit of backwards compatibility.

```suggestion
  questionnaireResponse?: QuestionnaireResponse
```